network/src/lib.rs

```diff
@@ -148,6 +148,38 @@ where
 	}
 }
 
+pub struct DelayedBlockAnnounceValidator<B: BlockT>(Arc<Mutex<Option<Box<dyn BlockAnnounceValidator<B> + Send>>>>);
```

Comment:
Docs and especially why we do this! + an issue that we need to fix this.

Reply:
done

Reply:
You should create an issue for this. And this should contain this info.

Reply:
Ok! Sorry, created #104

Reply:
done